src/OrbitGl/FrameTrackOnlineProcessor.h

```diff
@@ -22,13 +22,11 @@ class FrameTrackOnlineProcessor {
   void ProcessTimer(const orbit_client_protos::TimerInfo& timer_info,
                     const orbit_client_protos::FunctionInfo& function);
 
-  void AddFrameTrack(const CaptureData& capture_data,
-                     const orbit_client_protos::FunctionInfo& function);
-  void RemoveFrameTrack(const CaptureData& capture_data,
-                        const orbit_client_protos::FunctionInfo& function);
+  void AddFrameTrack(uint64_t function_id);
+  void RemoveFrameTrack(uint64_t function_id);
 
  private:
-  absl::flat_hash_set<uint64_t> current_frame_track_functions_;
+  absl::flat_hash_set<uint64_t> current_frame_track_function_ids_;
   absl::flat_hash_map<uint64_t, uint64_t> previous_timestamp_ns_;
```

Comment:
I think it would be beneficial if the name of this field explained what is mapping from and what it's mapping to, it's a bit cryptic like this. E.g., `function_id_to_previous_timestamp_ns_` or `previous_timestamp_ns_of_function_id_`.

Reply:
renamed